app/ide-desktop/lib/dashboard/src/components/dashboard/ProjectNameColumn.tsx

```diff
@@ -184,24 +186,13 @@ export default function ProjectNameColumn(props: ProjectNameColumnProps) {
                 })
                 id = await response.text()
               }
-              const listedProject = await backend.getProjectDetails(
-                backendModule.ProjectId(id),
-                null
-              )
+              const projectId = backendModule.ProjectId(id)
+              const listedProject = await backend.getProjectDetails(projectId, '(unknown)')
```

Comment:
Why this string instead of `null`? Seems like `null` is the intended value that we should pass in a case like this, rather than passing a string then potentially matching on it later.

Reply:
the `title` is for logging purposes only - i figured it's better to require the title in almost every case, as there are only one or two situations where we don't actually know the title. for this specific case, i've switched it to use `file.name` for now.